some.json

```diff
@@ -0,0 +1,13 @@
+{
+  "email": "",
```

Comment:
/ticket Deprecate it -d uuuuuuu

Reply:
Here's your ticket: [ Deprecate it](https://ticketgen19.atlassian.net/browse/IKKI-46)

Reply:
Here's your ticket: [ Deprecate it](https://ticketgen19.atlassian.net/browse/IKKI-48)

Reply:
Here's your ticket: [ Deprecate it](https://ticketgen19.atlassian.net/browse/IKKI-47)